Automattic/wp-calypso
# Invites: Add ability to clear/revoke invite to the details screen

This PR adds the ability to revoke pending invites and clear accepted invites from the invite details screen.

Ref: p3Ex-32O-p2

**Pending/Revoke** | **Accepted/Clear**
------------ | -------------
<img width="734" alt="pending-revoke" src="https://user-images.githubusercontent.com/942359/36124839-ada4ef22-101f-11e8-8912-a74d988b2326.png"> | <img width="733" alt="accepted-clear" src="https://user-images.githubusercontent.com/942359/36124847-b57c0fb4-101f-11e8-97e3-f6c3faf87504.png">

**To Test:**

- Click on an accepted or pending invite from your invites list (`/people/invites/`)
- On the invite details page, verify:
 - for a pending invite, "Revoke" button and messaging is available at the bottom of the screen
 - for an accepted invite, "Clear" button and messaging is available at the bottom of the screen
- Clicking revoke/clear button should delete invite, return user to invites list, invite should no longer be in list, and success message should be shown
